src/components/BTextarea/BTextarea.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div :class="classes">
+    <label v-if="!hideLabel" :for="id">
+      {{ label }}<span v-if="required" aria-label="Required">*</span>
+    </label>
+    <textarea
+      v-model="inputValue"
+      v-bind="inputAttributes"
+    />
+    <div class="b-input-group--error">
+      <span></span>
+    </div>
+  </div> 
+</template>
+
+<script>
+export default {
+  name: 'b-textarea',
+  props: {
+    id: {
+      type: String,
+      required: true
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    cols: {
+      type: Number,
+      default: 30
+    },
+    rows: {
+      type: Number,
+      default: 5
+    },
+    placeholder: String,
+    hideLabel: Boolean,
```

Comment:
Based on our conversation, I don't think there's a valid use case for a text area with no visible label so we should remove this option.